### ISSUE TITLE
riscv64-unknown-linux-gnu

### ISSUE THREAD

**Issue by NccChild** [xvisor/xvisor#162]
Hi,

I had an error when compiling with make command
![34aa68fc2a8491a2670626414cce17f4](https://github.com/xvisor/xvisor/assets/120999952/3986e9bf-2232-4ce8-9993-2fb8908cafe6)

I wonder why this error occurred, should I install some dependencies? But, I followed the README instruction in xvisor/docs/riscv/riscv64-qemu.txt. But  in step 4 when I make,i have some probelm

Thanks

**Comment by avpatel** [comment]
There is missing dependency for ./tools/scripts/d2c.py

**Comment by NccChild** [comment]
yes ,i meet this problem:d2c.py not found.But i install `python-is-python3` package.And this problem is solve,and now i can success run `python --version`and `python3 --version`. i don't know how to slove the problem.Can you tell me the specific steps to solve it？

**Comment by NccChild** [comment]
> There is missing dependency for ./tools/scripts/d2c.py

yes ,i meet this problem:d2c.py not found.But i install python-is-python3 package.And this problem is solve,and now i can success run python --versionand python3 --version. i don't know how to slove the problem.Can you tell me the specific steps to solve it？
